### DISCUSSION TITLE
Reflections on using Zettlr in combination with reesarch literature

### DISCUSSION THREAD

**Discussion by BabakFarshchian** [Zettlr/Zettlr#4975] [Show and tell]
Hi everyone. I wrote a blog post on how Zettlr has helped me with digging into research literature. Hope you will find it useful. And don't hesitate sharing your own experience/thoughts about the topic.

https://blog.babak.no/2024/02/19/how-zettlr-changed-my-relation-to-research-literature/

**Comment by Myarlathotep** [comment]
Thanks for sharing! I am just starting out using Zettlr as my knowledge and idea management system. I really like your system of having note types, I think that's something I want to implement to. A couple of questions I've got:

- How do you manage files using your note typing system? Do you have directories grouped for types? Or something else, like ideas/topics?
- Do you utilise the yaml front matter property of notes to define note types? Or some other method of stating that "this note is an idea", or whatever it may be?
- Do you have any advice for someone starting out using Zettlr for this style of knowledge management? 

Recently I've been yearning for a way to use Zettlr links to point to specific points in a file, rather than just an entire file. But reading your methodology, I get the feeling that it would be better to break down things I want to link to into specific notes, rather than amalgamated into a larger note. 

For instance with definitions, I wanted some way of having a "Definition" note containing all definitions, but instead it sounds like you have a note per definition, right?

**Reply by BabakFarshchian** [reply]
Thanks @Myarlathotep. Interesting to see you have been thinking of one definition note for all your definitions. I think the advantage would be that you will see all of them in one place, and maybe do some outlining and reordering and analysis when all of them are in one place. You can even have nested definitions by using different heading levels. There are definitely opportunities here!

I used one definition per note mainly because I wanted to use tags and links in Zettlr to build a Zettelkasten-type of database. I think the advantage is that it aligns with the functionality offered by Zettlr. For instance, in related notes window, you can see other related definitions when you open one definition note. I think this can be difficult if you go for one note for all definitions.

To answer your other questions, I don't use directories at all and rely on tags and links and search for everything I do in Zettlr. So I have a flat directory with all the notes in it (around 600 of them by now). I use YAML tags and have created templates for each node type that pre-fills them with the right tags for the node type.  I have no other advice than just use it, try and revise. I think organizing your Zettlr is a very personal thing so you need to see what works for you. Cheers!

**Reply by Myarlathotep** [reply]
Awesome, thanks Babak!

**Reply by digbyphonic** [reply]
excellent post and references.
very much appreciated.
my experience, progress, and software choices are very similar to yours.
i recently found Zettlr / Zettelkasten with the goal of re-structuring my note-taking / annotating / research / writing methods to create exactly the tool you describe.
your post is extremely encouraging --- i shall continue!
i have not made a note-type category, not gotten that far.
i'll re-visit your note-type suggestion in the near future.
i'm tagging as i go along and not keeping close track...at some point  patterns may emerge through back-linking and tagging and i'll streamline it.

as you say, it's a personalised process that refines as you progress.

thank you.
best wishes to you and your continued good work.

p.s. please allow me to point out small typo's you may wish to correct:
'write-click' / 'right-click'?
'concept-centric'

**Reply by BabakFarshchian** [reply]
Thank you @digbyphonic for you kind words I am happy you found this helpful, and thanks for pointing out the typos. 

My node types are probably version 20 of an ever evolving system, and most certainly not the last one! Good luck with building your system.
